### PR TITLE
Fix support for armv7l

### PR DESCRIPTION
**Pull request by seibert**

### CODE REVIEW

File: llvmlite/tests/test_binding.py [hunk 1]
```diff
@@ -18,6 +18,14 @@
 from . import TestCase
 
 
+# arvm7l needs extra ABI symbols to link successfully
+if platform.machine() == 'armv7l':
+    libgcc_s = ctypes.cdll.LoadLibrary('libgcc_s.so.1')
+    llvm.add_symbol('__aeabi_unwind_cpp_pr0',
```
Comment (pitrou) [inline llvmlite/tests/test_binding.py]: Did you try with `load_library_permanently` as suggested by Siu?

Comment (seibert) [inline llvmlite/tests/test_binding.py]: Oh, I missed that suggestion.  I'll try that next.

**Comment by seibert** [comment]
OK, using `load_library_permanently` seems to work, and does not require listing each symbol individually.

**Comment by pitrou** [comment]
This looks ok to me. Is something else needed in this PR for ARM support?

**Comment by seibert** [comment]
No other changes needed.  This can be merged.

**Pull request merged**